src/loader/doc_attr.ml

```diff
@@ -46,9 +46,15 @@ let attached parent attrs =
   let rec loop first nb_deprecated acc
       : _ -> (Model.Comment.docs, Model.Error.t) result =
     function
+#if OCAML_MAJOR = 4 && OCAML_MINOR >= 08
+    | {Parsetree.attr_name = { Location.txt =
+          ("doc" | "ocaml.doc"); loc = _loc}; attr_payload; _ } :: rest -> begin
+        match load_payload attr_payload with
+#else
     | ({Location.txt =
           ("doc" | "ocaml.doc"); loc = _loc}, payload) :: rest -> begin
         match load_payload payload with
+#endif
```

Comment:
You could limit the `#if .. #else .. #endif` to the pattern part if you make sure to use the same variable names on both sides.

Reply:
True, that'll probably look nicer. Thanks!